src/components/Navbar.js

```diff
@@ -0,0 +1,16 @@
+import React from 'react';
+// eslint-disable-next-line
```

Comment:
- [ ] Kindly make sure to not disable Linters. Linters help us to write clean code.